test/exometer_SUITE.erl

```diff
@@ -417,7 +417,7 @@ test_status(_Config) ->
      {module, exometer_histogram},
      {status, enabled},
      {cache, 0},
-     {value, []},
+     {value, _},
```

Comment:
Sure, at the very least matching the names of the datapoints, if not the corresponding values. But in this case, the metric has just been created and not updated yet, so the values should be pretty easy to match as well.